### DISCUSSION TITLE
Bootloader for F103RCT6

### DISCUSSION THREAD

**Discussion by Mootronic** [mriscoc/Ender3V2S1#1365] [Q&A]
Hi there,

I had a motherboard with an F401 but wanted to have access to those fancy options AND laser on an Ender 3S1.
So I've changed the STM32 from the 401 (desoldering) to a 103. It work well, as I can flash it with a ST-Link V2 with no problems.

I asked Creality for the 103 bootloader but they answer is : nope. Is there somebody here who can give me a memory dump of a STM32103RCT6 board ? I would grandly appreciate if possible. I've tested some other bootloaders around the Internet but none of theme works with Marlin.

Thanks in advance.

**Comment by mriscoc** [comment]
Maybe people from this thread could have an idea: https://github.com/mriscoc/Ender3V2S1/discussions/954

**Reply by Mootronic** [reply]
Thank you for the fast reply. I will check this.
